### DISCUSSION TITLE
send keyboard character keystrokes directly to a Server in (get-hpovServer) regardless of the underlying OS

### DISCUSSION THREAD

**Discussion by ehsanijavad** [HewlettPackard/POSH-HPOneView#535] [Ideas]
Hi 
I got the my server with get-HPOVserver -xxx and Mounted the CD.
I want to install ESXi unattended. and configure the ip and subnet and .... that 's why:
Please let me know someone know how can I send keyboard character keystrokes directly to a Server  in (get-hpovServer) regardless of the underlying OS.
or any idea?

Thank you for help.

**Comment by ChrisLynchHPE** [comment]
This is not possible.  Keystrokes cannot be sent to the iLO via a scripting tool.  You will need to use an unattended method to deploy your OS, which would provide an answer file for the installer questions.

**Comment by ehsanijavad** [comment]
Hi Chris

Thank you for your answer.
